devel/libwebsockets/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; indent-tabs-mode: nil -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        warmcat libwebsockets 2.4.1 v
+categories          devel net
+platforms           darwin
+license             LPGL-2.1
+
+maintainers         nomaintainer
+
+description         Canonical libwebsockets.org websocket library
+
+long_description    \
+    Libwebsockets is a lightweight pure C library built to use minimal \
+    CPU and memory resources, and provide fast throughput in both directions \
+    as client or server.
+
+distname            v${version}
+worksrcdir           ${name}-${version}
+
+homepage            https://github.com/warmcat/libwebsockets
+master_sites        https://github.com/warmcat/libwebsockets/archive
+
+checksums           rmd160 846a9b70b19eae0e485f921e5f770edb16e42a0e \
+                    sha256 29414be4f79f6abc0e6aadccd09a4da0f0c431e3b5691f496acd081ae6a8240c
+
+cmake.out_of_source yes
+configure.args      -DLWS_WITHOUT_TESTAPPS=ON \
```

Comment:
You should use `configure.args-append` instead, otherwise the default flags which are needed to compile the software correctly will be removed (replaced).

Reply:
Updated in both libwebsockets and mosquitto, thank you!